crates/oxc_ast/src/ast/literal.rs

```diff
@@ -112,6 +112,25 @@ bitflags! {
     }
 }
 
+impl TryFrom<char> for RegExpFlags {
```

Comment:
Should this be placed elsewhere? Should we additionally implement `TryFrom<&str>`, extracting logic from the lexer?

Reply:
> Should this be placed elsewhere?
Seems fine.

> Should we additionally implement TryFrom<&str>, extracting logic from the lexer?
If there are repeated code, we can put them into `oxc_syntax`.